app/src/main/res/values/styles.xml

```diff
@@ -27,6 +27,10 @@
         <item name="android:textAppearance">?textAppearanceListItem</item>
     </style>
 
+    <style name="ClimateOperation">
```

Comment:
I don't think one item and only one usage warrants an extra style. Or is it planned with more items for more views?